outputs.tf

```diff
@@ -2,12 +2,32 @@ output "id" {
   value = "${null_resource.default.triggers.id}"
 }
 
+output "name" {
+  value = "${null_resource.default.triggers.name}"
+}
+
+output "namespace" {
+  value = "${null_resource.default.triggers.namespace}"
+}
+
+output "stage" {
+  value = "${null_resource.default.triggers.stage}"
+}
+
+output "attributes" {
+  value = "${null_resource.default.triggers.attributes}"
+}
+
+# Merge input tags with our tags.
+# Note: `Name` has a special meaning in AWS and we need to disamgiuate it by using the computed `id`
 output "tags" {
   value = "${
-    map(
-      "Name", "${null_resource.default.triggers.id}",
-      "Namespace", "${null_resource.default.triggers.namespace}",
-      "Stage", "${null_resource.default.triggers.stage}"
+    merge( 
```

Comment:
Let's add a note to the `README.md` regarding tags merge operation.
Because end users can make a mistake by adding tag with `Namesapce`, `Stage` or `Name` key.
We should add `Warning` regarding it.